source/help/reporting_issues.markdown

```diff
@@ -4,36 +4,42 @@ description: "Reporting issues about Home Assistant"
 
-Please refer to the [**Settings** -> **About**](https://my.home-assistant.io/redirect/info/) page, which is accessible inside the Home Assistant frontend.
+To see the version information, refer to the [**Settings** > **About**](https://my.home-assistant.io/redirect/info/) page.
 
-## First Home Assistant release with the issue
+## Description of the problem
 
-Please provide the release which contains the issue.
+Provide a summary of your issue and tell us what's wrong. Tell us what you were trying to do and what happened.
+
+There are integrations which require additional steps (installing third-party tools, compilers, etc.) to get your setup working. Please describe the steps you took and the ones to reproduce the issue if needed.
+
+## First Home Assistant version with the issue
+
+Please provide the version which contains the issue. You can find the version information under [**Settings** > **System** > **Repairs** > Three dots in the upper right > **System information**](https://my.home-assistant.io/redirect/system_health/).
 
 ## Last working Home Assistant release (if known)
 
-If possible, provide the latest release of which you know that the integration or platform was working. Home Assistant is evolving very fast and issues may already be addressed or be introduced by a recent change.
+If possible, provide the latest release of which you know that the integration or platform was working. Home Assistant is evolving very fast and issues may already be addressed or be introduced by a recent change. You can find the detailed information about your system under [**Settings** > **System** > **Repairs** > Three dots in the upper right > **System information**](https://my.home-assistant.io/redirect/system_health/)
 
 ## Operating environment
 
-There are many different ways to run Home Assistant. In this section please mention which you are using, e.g., Home Assistant (using the Home Assistant Operating System), Home Assistant Supervised, Home Assistant Core in Docker or a manual installation of the Home Assistant Core. It can help if you mention which operating system you are using because not all are supported on the same level.
+There are many different ways to run Home Assistant. In this section, please mention which you are using: Home Assistant Operating System, Home Assistant Supervised, Home Assistant Core in Docker or a manual installation of the Home Assistant Core. It can help if you mention which operating system you are using because not all are supported on the same level. You can find this information under [**Settings** > **System** > **Repairs** > Three dots in the upper right > **System information**](https://my.home-assistant.io/redirect/system_health/).
```

Comment:
```suggestion
There are many different ways to run Home Assistant. In this section, please mention which you are using: Home Assistant Operating System, Home Assistant Supervised, Home Assistant Core in Docker, or a manual installation of the Home Assistant Core. It would be helpful to mention which operating system you are using because not all are supported on the same level. You can find this information under {% my system_health title="**Settings** > **System** > **Repairs** > Three dots in the upper right > **System information**" %}.
```